config-windows.md

```diff
@@ -40,15 +40,15 @@ The following parameters can be specified:
 
 * **`shares`** *(uint16, OPTIONAL)* - specifies the relative weight to other containers with CPU shares. The range is from 1 to 10000.
 
-* **`percent`** *(uint, OPTIONAL)* - specifies the percentage of available CPUs usable by the container.
+* **`maximum`** *(uint, OPTIONAL)* - specifies the portion of processor cycles that this container can use as a percentage times 100 . Range is from 1 to 10000.
```

Comment:
> I think we want the change from percent to maximum, but we dont want to encode the limit into the specs.

The important exception to this, discussed in [today's meeting][1], is when the value is unique to this spec and there are no matching kernel docs.  I expect this property, and likely `shares` ([adjusted][2] in #780) both fall into that category, and need to retain their runtime-spec-defined limits.

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2017/opencontainers.2017-04-26-15.00.html
[2]: https://github.com/opencontainers/runtime-spec/pull/780/files#diff-36e882271af2f3ad47923159cb746598L41